common/static/js/cases_search_in_case.js

```diff
@@ -0,0 +1,172 @@
+/*
+Search for labels and text content in case UI.
+*/
+
+function Searchable({text, childElements, targetUrl, targetLabel, targetPageName} = {}) {
+  this.text = text
+  this.childElements = childElements
+  this.targetUrl = targetUrl
+  this.targetLabel = targetLabel
+  this.targetPageName = targetPageName
+}
+
+let searchables = []
```

Comment:
I'm 90% sure this can be a const

Reply:
Yup. That works. df3b9bdc981b8f43464f965d2eea08823955ad9f